evalml/tests/model_understanding_tests/prediction_explanations_tests/test_force_plots.py

```diff
@@ -0,0 +1,279 @@
+        for result in results:
+            for class_label in expected_class_labels:
+                assert isinstance(
+                    result[class_label]["plot"],
+                    shap.plots._force.AdditiveForceVisualizer,
+                )
+
+
+@pytest.mark.parametrize(
+    "rows_to_explain, just_data", product([[0], [0, 1, 2, 3, 4]], [False, True])
+)
+def test_force_plot_multiclass(rows_to_explain, just_data, has_minimal_dependencies):
+    if has_minimal_dependencies:
+        pytest.skip("Skipping because plotly not installed for minimal dependencies")
+
+    X, y = load_wine()
+
+    pipeline = MulticlassClassificationPipeline(
+        component_graph=["Simple Imputer", "Random Forest Classifier"]
+    )
+    pipeline.fit(X, y)
+
+    if just_data:
+        results = force_plot(pipeline, rows_to_explain, X, y)
+    else:
+        results = graph_force_plot(
+            pipeline,
+            rows_to_explain=rows_to_explain,
+            training_data=X,
+            y=y,
+            matplotlib=False,
+        )
+
+    # Should have a result per row to explain.
+    assert len(results) == len(rows_to_explain)
+
+    expected_class_labels = {"class_0", "class_1", "class_2"}
+
+    for result in results:
+        class_labels = result.keys()
+        assert len(class_labels) == len(set(y))
+        assert set(class_labels) == expected_class_labels
+        for class_label in expected_class_labels:
+            assert {"expected_value", "feature_names", "shap_values"}.issubset(
+                set(result[class_label].keys())
+            )
+
+    if not just_data:
+        # Should have a force plot for each row result.
+        for result in results:
+            for class_label in expected_class_labels:
+                assert isinstance(
+                    result[class_label]["plot"],
+                    shap.plots._force.AdditiveForceVisualizer,
+                )
+
+
+@pytest.mark.parametrize(
+    "rows_to_explain, just_data", product([[0], [0, 1, 2, 3, 4]], [False, True])
+)
+def test_force_plot_regression(
+    rows_to_explain, just_data, X_y_regression, has_minimal_dependencies
+):
+    if has_minimal_dependencies:
+        pytest.skip("Skipping because plotly not installed for minimal dependencies")
+
+    X, y = X_y_regression
+    X = pd.DataFrame(X)
+    y = pd.Series(y)
+
+    pipeline = RegressionPipeline(
+        component_graph=["Simple Imputer", "LightGBM Regressor"]
+    )
+    pipeline.fit(X, y)
+
+    if just_data:
+        results = force_plot(pipeline, rows_to_explain, X, y)
+    else:
+        results = graph_force_plot(
+            pipeline,
+            rows_to_explain=rows_to_explain,
+            training_data=X,
+            y=y,
+            matplotlib=False,
+        )
+
+    # Should have a result per row to explain.
+    assert len(results) == len(rows_to_explain)
+
+    for result in results:
+        class_labels = result.keys()
+        assert len(class_labels) == 1
```

Comment:
why do we assert `len(class_labels)==1` here? Does this imply that there's only 1 class we're doing this for on regression pipelines?